.travis.yml

```diff
@@ -23,4 +23,4 @@ script:
   - cd $TRAVIS_BUILD_DIR
   # Travis windows is changing line endings, breaking tests. Switch to LF.
   - find langserver -type f -print0 | xargs -0 sed -i ':a;N;$!ba;s/\r//g'
-  - go test -timeout 5m -race ./...
+  - go test -short -timeout 5m -race ./...
```

Comment:
so we add this flag to avoid the heavy integration tests in go-buildserver?

Reply:
Yup